site/pages/type-of-consequence.test.tsx

```diff
@@ -0,0 +1,31 @@
+import renderer from "react-test-renderer";
+import { describe, it, expect } from "vitest";
+import TypeOfConsequence from "./type-of-consequence.page";
+import { ConsequenceType, TransportMode } from "../constants/enum";
+import { AddConsequenceProps, ErrorInfo } from "../interfaces/index";
+
+const noErrors: ErrorInfo[] = [];
+const withNoInputs: AddConsequenceProps = {};
+
+const withErrors: ErrorInfo[] = [
+    { id: "consequenceType", errorMessage: "Select a consequence type" },
+    { id: "modeOfTransport", errorMessage: "Select a mode of transport" },
+];
+const withInputs: AddConsequenceProps = {
+    modeOfTransport: TransportMode.bus,
+    consequenceType: ConsequenceType.networkWide,
+};
+
+describe("pages", () => {
+    describe("CreateDisruption", () => {
+        it("should render correctly with no inputs", () => {
+            const tree = renderer.create(<TypeOfConsequence errors={noErrors} inputs={withNoInputs} />).toJSON();
+            expect(tree).toMatchSnapshot();
+        });
+
+        it("should render correctly with inputs", () => {
+            const tree = renderer.create(<TypeOfConsequence errors={withErrors} inputs={withInputs} />).toJSON();
+            expect(tree).toMatchSnapshot();
+        });
+    });
```

Comment:
The first test is no errors and no inputs, the second test is with inputs AND errors - can we change the name of the test to reflect this please?

We also need a third test, for inputs but no errors.

Reply:
done